SPECS-EXTENDED/perl-Razor-Agent/perl-Razor-Agent.spec

```diff
@@ -0,0 +1,314 @@
+Summary:        Use a Razor catalogue server to filter spam messages
+Name:           perl-Razor-Agent
+Version:        2.85
+Release:        37%{?dist}
+License:        Artistic 2.0
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+URL:            https://razor.sourceforge.net/
+Source:         %{_mariner_sources_url}/razor-agents-%{version}.tar.bz2
```

Comment:
Is the upstream dead? Please add a short comment explaining why we have to use our blob storage.